api/SVGElement.json

```diff
@@ -347,6 +347,54 @@
           }
         }
       },
+      "nonce": {
+        "__compat": {
+          "spec_url": "https://html.spec.whatwg.org/multipage/urls-and-fetching.html#dom-noncedelement-nonce",
```

Comment:
@sideshowbarker can you tell why the domintro block just above this doesn't have an ID?

Reply:
> @sideshowbarker can you tell why the domintro block just above this doesn't have an ID?

I don’t know — but I’ll take a look at the spec source and what wattsi is doing with it, and figure out if I can manage to get an ID generated for it.

Reply:
Thanks! I'll go ahead and merge it like this in the meantime.